package.json

```diff
@@ -1,63 +1,53 @@
 {
-  "name": "name-your-contributors",
-  "version": "0.0.0-development",
-  "description": "Name your GitHub contributors; get commits, issues, and comments",
-  "license": "MIT",
-  "repository": "RichardLitt/name-your-contributors",
-  "author": {
-    "name": "Richard Littauer",
-    "email": "richard.littauer@gmail.com",
-    "url": "burntfen.com"
-  },
-  "bin": "cli.js",
-  "scripts": {
-    "lint": "standard",
-    "test": "standard && ava",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
-  },
-  "keywords": [
-    "cli-app",
-    "cli",
-    "name",
-    "contribs",
-    "contrib",
-    "contributor",
-    "contributors",
-    "github",
-    "OSS",
-    "open source",
-    "commenters",
-    "creators",
-    "reviewers",
-    "review",
-    "code review",
-    "community",
-    "contribute",
-    "contributing"
-  ],
-  "dependencies": {
-    "ava": "^0.22.0",
-    "bluebird": "^3.4.1",
-    "depaginate": "0.0.8",
-    "format-gh-users": "^1.0.2",
-    "get-github-user": "0.2.1",
-    "gitconfiglocal": "^2.0.1",
-    "lodash": "^4.14.2",
-    "meow": "^3.7.0",
-    "mkdirp": "^0.5.1",
-    "moment": "^2.14.1",
-    "octokat": "^0.9.2",
-    "pify": "^3.0.0",
-    "sort-alphabetic": "0.0.0"
-  },
-  "devDependencies": {
-    "eslint": "^4.6.1",
-    "standard": "^10.0.3",
-    "xo": "^0.18.2",
-    "semantic-release": "^8.0.0"
-  },
-  "coordinates": [
-    45.5231596,
-    -73.5870259
-  ]
+	"name": "name-your-contributors",
+	"version": "0.0.1",
```

Comment:
Leave it as 0.0.0-development. Using [semantic-release](https://github.com/semantic-release/semantic-release).